arc-script/arc-script-core/src/compiler/ast/from/parse/error.rs

```diff
@@ -1,7 +1,7 @@
 use crate::compiler::ast;
 use crate::compiler::ast::from::parse::lexer::Token;
 use crate::compiler::info::diags::{Diagnostic, Error};
-use crate::compiler::info::files::{FileId, Loc, ByteIndex};
+use crate::compiler::info::files::{ByteIndex, FileId, Loc};
```

Comment:
Does this belong to this commit?

Reply:
Not sure how that made it in, but I undid it now :)

Reply:
A second pair of eyes is allways good.